src/providers/ipa/selinux_child.c

```diff
@@ -282,11 +282,6 @@ int main(int argc, const char *argv[])
      *   [main] (0x0020): Cannot set SELinux login context.
      *   [main] (0x0020): selinux_child failed!
      */
-    if (unsetenv("_SSS_LOOPS") != 0) {
-        DEBUG(SSSDBG_CRIT_FAILURE,
-              "Failed to unset _SSS_LOOPS, some libsemanage functions might "
-              "fail.\n");
-    }
```

Comment:
Why is this removed?

Reply:
Reintroducing the lines after the IRC discussion.